server/config/index.js

```diff
@@ -57,7 +57,12 @@ const config = {
     type: null, // e.g. type 'keycloak' or 'passport'
     config: null // implementation specific config
   },
-  schemaListenerConfig: undefined
+  schemaListenerConfig: undefined,
+  serverSecurity: {
+    introspection: process.env.INTROSPECTION || true,
```

Comment:
I actually think we can totally remove the introspection stuff altogether. One thing you will notice is if you set `NODE_ENV=production` then introspection and the playground are automatically disabled.

Reply:
What about the case where the developer wants to disable introspection (for whatever reason) and run a dev environment?

Reply:
@StephenCoady that developer could run it in their dev environment with `NODE_ENV=production` 😄

Reply:
Ah ok, I didn't realise we weren't really doing anything else differently. Will push that now.